src/mapped_nca.cpp

```diff
@@ -298,4 +303,38 @@ MappedNca::parse(std::unique_ptr<mk::mem::MappedData> backing_data,
 
   return nca;
 }
+
+NcaSectionView::NcaSectionView(const MappedNca &nca, int section)
+    : _nca(nca), _section(section) {
+  _current_section = reinterpret_cast<uint8_t *>(malloc(BLOCK_SIZE_BYTES));
```

Comment:
casting the return of malloc without a sizeof directive 😱 I'm not sure which side you fall on!

Reply:
I mean, this is c++ so I can't implicit cast void* to uint8_t*, so I have to cast. Not sure what you're getting at re: sizeof? `malloc` allocates `size` _bytes_, and a uint8_t better be exactly one byte